internal/game/repository.go

```diff
@@ -1,5 +1,28 @@
 package game
 
+import (
+	"database/sql"
+	"encoding/json"
+)
+
+type NullInt64 struct {
+	sql.NullInt64
+}
```

Comment:
Why are we using NullInt64 though? 🤔

Reply:
because requests body are dynamic, they can be null, this helps us to handle null cases

Reply:
alright nice